out/HtmlBunPlugin.d.ts

```diff
@@ -1,16 +1,19 @@
-import type { BunPlugin } from 'bun';
+import type { BunPlugin } from "bun";
+
 export interface HtmlBunPluginConfig {
-    // something
-    filename: string
-    title?: string;
-    template?: string;
+  filename: string;
+  title?: string;
+  template?: string;
+  publicPath?: string;
 }
 /**
  *
  * @param {HtmlBunPluginConfig} config - optional options object
  * - filename: Defaults to 'index.html'. Required if passing in an options object.
  * - title?: adds a title tag with the provided string. Defaults to 'Bun App'
  * - template?: path to a template HTML file to inject entries and title into
+ * - publicPath?: the base path that the entrypoint will be fetched from, defaults to './'
  */
 declare function HtmlBunPlugin(config?: HtmlBunPluginConfig): BunPlugin;
+
 export default HtmlBunPlugin;
```

Comment:
This file wasn't automatically updated for me, I'm not sure why, so I updated it manually. Is that correct process?

Reply:
I also realized I didn't specify a build process for this when I saw this PR. I believe I just ran the TS compiler to generate only d.ts files the first time around.

After merging this PR, I'm going to add a `HtmlBunPlugin.d.ts` file to `src/`, and have it copied over to `out/` during the build process. It'll be a good-enough solution for now.